languages/kotlin/highlights.scm

```diff
@@ -263,6 +263,16 @@
 ]@keyword
 
 [
+	"if"
```

Comment:
I really hate to be that guy, but it looks like new keywords use spaces for indent, whereas everything else uses tabs.

Reply:
I honestly wouldn't worry too much about it. If I were you, I'd just `git commit --fixup sha_of_original_commit` and call it a day.

Reply:
We generally tend to avoid that conversation altogether and use a standardized formatter like rustfmt.

Reply:
We use spaces, not tabs. Could you put the spaces back, please?